nebula/api.go

```diff
@@ -104,18 +104,27 @@ func (c *APIClient) TryUpdate(siteName string, hostID string, privateKey string,
 	// Check for update
 	ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
```

Comment:
Is the anticipation to actually poll or to use the api for its side effects? 30s doesn't match what normal dnclients use for their timeouts but I also don't see this actually maintaining a http request, it fires based on current timers which iirc was 15 minutes or so.

Reply:
We will get 409 client gone away messages if the client leaves early. If we plan to use LongPollWait without passing the immediate flag, I believe the intention is to complete the full request, so we should make sure the timeout is longer than 60 seconds, which is the API long poll window.